spec/mongoid/interceptable_spec.rb

```diff
@@ -1835,12 +1841,63 @@ class TestClass
       end
     end
 
+    context "embeds_one" do
+      let(:registry) { InterceptableSpec::CallbackRegistry.new }
+
+      let(:parent) do
+        InterceptableSpec::CbEmbedsOneParent.new(registry).tap do |p|
+          p.child = InterceptableSpec::CbEmbedsOneChild.new(registry)
```

Comment:
If you don't mind, `s/p/parent/` please, to save us debugging grief later. I see that I probably started with `p` as the variable name here which was a poor choice on my part.